packages/web/src/coh/commanders.ts

```diff
@@ -10,17 +11,15 @@ const convertCommanderIDToName = (commanderID: number): string => {
     }
 };
 
-const getCommanderData = (commanderID: string): Record<string, any> => {
+const getCommanderData = (commanderID: string): CommanderData | null => {
     if (Object.prototype.hasOwnProperty.call(allCommanders, commanderID)) {
         return allCommanders[commanderID];
     } else {
-        return {};
+        return null;
     }
 };
 
-const getCommanderByRaces = (
-    raceName: "wermacht" | "usf" | "soviet" | "wgerman" | "british",
-): Array<Record<string, any>> => {
+const getCommanderByRaces = (raceName: RaceName): Array<CommanderData> => {
```

Comment:
Tady zmena misto  `"wermacht" | "usf" | "soviet" | "wgerman" | "british"` jsem si definovaly datovy typ ktery definguje primo takovy string. Takze to nemusime kopirovat amuzeme to zmenit jen na jednom miste. 

A dalsi vec, vraci to pole commanderData misto random Record string